frontend/lib/src/components/shared/AlertContainer/AlertContainer.test.tsx

```diff
@@ -30,27 +32,17 @@ const getProps = (
 
 describe("AlertContainer element", () => {
   it("renders a Notification", () => {
-    const wrapper = mount(<AlertContainer {...getProps()}></AlertContainer>)
-
-    expect(wrapper.find("Notification").exists()).toBeTruthy()
+    render(<AlertContainer {...getProps()} />)
+    expect(screen.getByTestId("stNotification")).toBeInTheDocument()
   })
 
   it("renders its children", () => {
-    const wrapper = mount(
+    render(
       <AlertContainer {...getProps()}>
-        <div className="foo" />
+        <div className="foo" data-testid="foo" />
       </AlertContainer>
     )
 
-    expect(wrapper.find(".foo").exists()).toBeTruthy()
-  })
-
-  it("sets its width", () => {
```

Comment:
deleted this test but it should be fine as we have e2e test coverage. In addition, tried grabbing all of the div content in this test and all of the divs have `""` width.

Reply:
Most likely would have to pass a `data-testid` as prop to the Body override that has the width set to accomplish this, but if we have a couple e2e tests that set width then that works too!

Reply:
hm I think the PR already passes a `data-testid` as a prop?

Reply:
Whoops 😅